internal/pkg/cli/flag.go

```diff
@@ -225,6 +226,10 @@ Must be either "MySQL" or "PostgreSQL".`
 Tasks with the same group name share the same set of resources. 
 (default directory name)`
 	taskImageTagFlagDescription = `Optional. The container image tag in addition to "latest".`
+	generateCMDFlagDescription  = `Optional. Given a non-Copilot ECS service or a Copilot service, generate a command with pre-filled values for each flags.
```

Comment:
```suggestion
	generateCMDFlagDescription  = `Optional. Generate a command with pre-filled values for each flag.
```
Just a suggestion. We probably don't need "Given ... service" because it's clear from the next two lines?